lib/version_utils.pm

```diff
@@ -973,3 +974,12 @@ sub is_community_jeos {
     return (get_var('FLAVOR', '') =~ /JeOS-for-(AArch64|RISCV|RPi)/);
 }
 
+=head2 has_selinux_by_default
+
+Returns true if the distro has SELinux as default MAC
+=cut
+
+sub has_selinux_by_default {
+    return is_tumbleweed || is_micro;
```

Comment:
And probably incorporate the changes from @volodymyrkatkalov: https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/e0b8fb66611c5a4806d612e2e67ff48aa6673456

Reply:
added except for `get_var('SELINUX', 0) || ` as this should be a seperate interface probably